Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -575,6 +575,10 @@ frida_code = """
                 // Create an immutable ObjC string object from a JS string object.
                 var str_url = NSString.stringWithString_(myNSURL.toString());
                 NSLog(str_url);
+
+                // Print the URL to the standard console
+                console.log(str_url);
```

Comment:
mayb eexplain differnece between console.log and NSLog here?

Reply:
I've enhanced the comments to explain that